eservice-descriptors-archiver/src/main/resources/application.conf

```
@@ -0,0 +1,21 @@
+eservice-versions-archiver {
```

Rememeber to update also references in `ApplicationConfiguration`
```suggestion
eservice-descriptors-archiver {
```